ini-templates/base-template.ini

```diff
@@ -74,7 +74,7 @@ set remote_indexing_threshold = ${remote_indexing_threshold}
 use = egg:encoded#indexer
 app = app
 path = /index_vis
-timeout = 60
+set timeout = 60
```

Comment:
these can't change.  
- 'set timeout = #3' allows us to read the timeout in the config in the app
- 'timeout = #' is what the es_indexer_listener has access too.